packages/common/package.json

```diff
@@ -39,6 +39,7 @@
     "@types/d3-dsv": "1.0.36",
     "@types/d3-ease": "1.0.8",
     "@types/d3-format": "1.3.1",
+    "@types/d3-hierarchy": "1.1.6",
```

Comment:
Should not be in common - the only d3 items in common are ones used by common...

Reply:
I thought we were going to stop importing d3 modules in any packages other than common? I think you said it had something to do with typing? Maybe I misunderstood?

Reply:
I don't believe I said that - there was an issue with d3-event having more than one instance?